portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,27 @@
 package com.google.sps.servlets;
 
+import java.util.*;
```

Comment:
Please include all imports explicitly: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

This simplifies dependency management by making it clearer which classes you actually depend on.

Reply:
I fixed this issue in my latest commit.